### DISCUSSION TITLE
Add TopicStats to DataPlatformIterableSource

### DISCUSSION THREAD

**Discussion by jhurliman** [foxglove/community#308] [Q&A]
https://github.com/foxglove/studio/blob/main/packages/studio-base/src/players/IterablePlayer/DataPlatformIterableSource.ts#L115-L116

**Comment by defunctzombie** [comment]
I've removed the bug label. A lack of topic stats is not a bug since topic stats are by their design a best effort bit of information (live connections don't provide them for all topics).
